npm_and_yarn/spec/dependabot/npm_and_yarn/file_fetcher_spec.rb

```diff
@@ -1787,7 +1787,37 @@
       expect(file_fetcher_instance.files.map(&:name)).
         to eq(%w(package.json package-lock.json .npmrc))
       expect(file_fetcher_instance.files.find { |f| f.name == ".npmrc" }.content).
-        to eq("registry=https://npm.fury.io")
+        to eq("registry=https://npm.fury.io/dependabot/")
```

Comment:
I don't think that this test was correct

Reply:
I created a test org to double check this 

![image](https://user-images.githubusercontent.com/8935151/230828009-b9ecffa7-cbf4-42b8-8546-58a717987e1b.png)

Reply:
Thanks for trying this in real life! 💪